build-in-container/mariner-docker-run.sh

```diff
@@ -0,0 +1,125 @@
+#! /bin/bash
+
+# The toolkit may not have created these yet, create them now so we can mount them
+mkdir -p ${repo_dir}/build/rpm_cache/cache
+mkdir -p ${repo_dir}/build/toolchain_rpms
+mkdir -p ${repo_dir}/ccache
```

Comment:
Can you add this to `.gitignore`?
(I've suggested updating the instructions to use the existing hello_world.spec, so these files will end up in the repo root where we already have ignores for most of the folders.

Reply:
(Also `logs` while you are at it).